src/Common/src/System/Security/Cryptography/Oids.cs

```diff
@@ -20,8 +20,12 @@ internal static partial class Oids
         internal const string Rsa = "1.2.840.113549.1.1.1";
         internal const string RsaOaep = "1.2.840.113549.1.1.7";
         internal const string RsaPss = "1.2.840.113549.1.1.10";
+        internal const string RsaPkcs1Sha256 = "1.2.840.113549.1.1.11";
+        internal const string RsaPkcs1Sha384 = "1.2.840.113549.1.1.12";
+        internal const string RsaPkcs1Sha512 = "1.2.840.113549.1.1.13";
         internal const string Esdh = "1.2.840.113549.1.9.16.3.5";
         internal const string EcDiffieHellman = "1.3.132.1.12";
+        internal const string Ecc = "1.2.840.10045.2.1";
```

Comment:
This is already here, as `EcPublicKey`

Reply:
So is DSA and RSA, they are there twice too. I can clean it up if you prefer that.

Reply:
Maybe just `Dsa` and `DsaPublicKey` actually. I thought `EcPublicKey` is inaccurate because it is also used for private keys.

Reply:
`id-ecPublicKey` is the name of that OID, so `EcPublicKey` is more appropriate.  Apparently when I made `DsaPublicKey` I was wrong (it's just `id-dsa`)

Reply:
Ok, will remove the duplicate ones then and use `EcPublicKey`.